com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -1538,23 +1535,8 @@ internal void HandleSceneEvent(ulong clientId, Stream stream)
                     SceneEventData.OnRead(reader);
                     NetworkReaderPool.PutBackInPool(reader);
 
-                    if (SceneEventData.SceneEventType == SceneEventData.SceneEventTypes.S2C_Sync)
-                    {
-                        // For this event the server may be sending scene event data about multiple scenes, so we need
-                        // to track a metric for each one.
-                        foreach (var sceneIndex in SceneEventData.ScenesToSynchronize)
-                        {
-                            m_NetworkManager.NetworkMetrics.TrackSceneEventReceived(
-                               clientId, (uint)SceneEventData.SceneEventType, ScenesInBuild[(int)sceneIndex], stream.Length);
-                        }
-                    }
-                    else
-                    {
-                        // For all other scene event types, we are only dealing with one scene at a time, so we can read it
-                        // from the SceneEventData directly.
-                        m_NetworkManager.NetworkMetrics.TrackSceneEventReceived(
-                           clientId, (uint)SceneEventData.SceneEventType, ScenesInBuild[(int)SceneEventData.SceneIndex], stream.Length);
-                    }
+                    m_NetworkManager.NetworkMetrics.TrackSceneEventReceived(
+                       clientId, (uint)SceneEventData.SceneEventType, ScenesInBuild[(int)SceneEventData.SceneIndex], stream.Length);
```

Comment:
Are we sure about the stream length? I feel like we had that issue at an earlier time and it was showing the full buffer size of the stream.

Reply:
That's a good point I haven't encountered this issue before, but this is also how I think all the other metrics I've implemented are working. Can we get this merged and raise a tech debt ticket open to check for this issue everywhere?